demo/index.html

```diff
@@ -14,39 +14,69 @@
   <title>marked-element demo</title>
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../../paper-styles/demo-pages.html">
   <link rel="import" href="../marked-element.html">
+
+  <style is="custom-style">
+
+    .centered {
+      max-width: 550px;
+      padding: 0;

```

Comment:
the inner content is wider than `400px`, but I added a `overflow:auto` in case the viewport width goes below 550px.  `vertical-section` has a `padding:24px`, so I reseted it in `.centered`. I know this is messy and just temporary, we may need a class to style certain demos in this way.